contracts/truefi/TrueLender.sol

```diff
@@ -189,8 +189,8 @@ contract TrueLender is ITrueLender, Ownable {
 
         minApy = 1000;
         maxApy = 3000;
-        participationFactor = 10000;
-        riskAversion = 15000;
+        minVotes = 15 * (10**6) * (10**8);
+        minYesToNoRatio = 4;
```

Comment:
set to 8000 = 80%

Reply:
even though technically this will never be called, it's fine to have it in case we deploy a new Lender